packages/object-schema/package.json

```
@@ -23,8 +23,9 @@
     "test": "tests"
   },
   "scripts": {
-    "build:prepend-type-ref": "node ../../tools/prepend-type-ref.js dist/esm/index.js",
-    "build": "rollup -c && tsc -p tsconfig.esm.json && tsc -p tsconfig.cjs.json && npm run build:prepend-type-ref",
+    "build:cts": "node -e \"fs.copyFileSync('dist/esm/index.d.ts', 'dist/cjs/index.d.cts')\"",
+    "build": "rollup -c && tsc -p tsconfig.esm.json && npm run build:cts",
```

```suggestion
    "build": "tsc -p tsconfig.esm.json && rollup -c && npm run build:cts",
```

moving `tsc` to the first, so we can just use `rollup-plugin-copy` - `build:cts` is no longer needed.

---

That doesn't work. Rollup generates the file that `tsc` evaluates.